### PR TITLE
Master version bump

### PR DESCRIPTION
**Pull request by sanderr**
- major version bump for master
- increment db schema v18 -> v30

### CODE REVIEW

**Review by wouterdb — APPROVED** [review]
This PR has the coolest title ever! 

Master version bump on core, pure science fiction

**Review by wouterdb — CHANGES_REQUESTED** [review]
Isn't 18->20 a bit tight, that is only two versions for ISO4?

**Comment by sanderr** [comment]
Fair point, I'll make it v30

**Comment by sanderr** [comment]
This is also according to the design, I misread it.

**Review by wouterdb — APPROVED** [review]
with remarks about DB version

File: src/inmanta/db/versions/v18.py [hunk 1]
```diff
@@ -1,27 +0,0 @@
-"""
```
Comment (arnaudsjs) [inline src/inmanta/db/versions/v18.py:1]: I think we should first get some more clarity about the DB versioning schema before start to remove these files. @wouterdb What do you think?

**Review by arnaudsjs — CHANGES_REQUESTED** [review]

File: src/inmanta/db/versions/v18.py (hunk 1 above)
Comment (sanderr) [reply src/inmanta/db/versions/v18.py:1]: I think we are in agreement on the design once again. Do you think it's fine to remove them now?

**Review by arnaudsjs — APPROVED** [review]

File: src/inmanta/db/versions/v18.py (hunk 1 above)
Comment (wouterdb) [reply src/inmanta/db/versions/v18.py:1]: I think we can now.

**Pull request merged**